strax/chunk.py

```diff
@@ -233,11 +236,35 @@ def concatenate(cls, chunks):
         data_type = data_types[0]
 
         run_ids = [c.run_id for c in chunks]
+        is_superrun = np.all([r.startswith('_') and not r.startswith('_temp_') for r in run_ids])
```

Comment:
> r in run_id is not going to be a list always so looping over it might not always work.

Hmm maybe it was late yesterday or I do not understand what you mean :D. It should always be a list see line above. 

> Also, can't you do simply is_superrun = subruns is not None

Yes you are right I will change it. Subruns just came later into the PR.

Reply:
Ah you are also right, so easy to mix up `run_id` and `run_ids`.